sites/next.skeleton.dev/src/content/docs/components/tab/react.mdx

```diff
@@ -0,0 +1,88 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import { Page as Example } from '@examples/components/tabs/Example.tsx';
+import ExampleRaw from '@examples/components/tabs/Example.tsx?raw';
+import { Page as IconsExample } from '@examples/components/tabs/IconsExample.tsx';
+import IconsExampleRaw from '@examples/components/tabs/IconsExample.tsx?raw';
```

Comment:
For all other components the example components are named "ExampleFoo" - with the word "example" at the start. I suggest we match that convention.